drivers/pwm/pwm_rcar.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2022 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_rcar_pwm
+
+#include <errno.h>
+#include <device.h>
+#include <devicetree.h>
+#include <soc.h>
+#include <drivers/pwm.h>
+#include <drivers/clock_control.h>
+#include <drivers/clock_control/rcar_clock_control.h>
+#include <drivers/pinctrl.h>
+
+#define LOG_LEVEL CONFIG_PWM_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(pwm_rcar);
+
+/* Helper macros for PWM */
+#define DEV_PWM_CFG(dev)        ((const struct pwm_rcar_cfg *)(dev)->config)
+#define DEV_PWM_DATA(dev)       ((struct pwm_rcar_data *)(dev)->data)
+
+/* PWM Controller capabilities */
+#define RCAR_PWM_MAX_CYCLE      1023U
```

Comment:
can you align all the constant definition ?

Reply:
Uncrustify matted ! haha ;) thanks !